main/ser2net/APKBUILD

```diff
@@ -43,25 +33,13 @@ build() {
 }
 
 package() {
-	cd "$_builddir"
+	cd "$builddir"
 	make DESTDIR="$pkgdir" install || return 1
```

Comment:
while you are at it, maybe also remove all the  ` || return 1` from the APKBUILD?